### PR TITLE
Add new midstream branches and update pipelines

### PR DESCRIPTION
**Pull request by serverless-qe**
Discover new branches and update Konflux pipelines using openshift-knative/hack.

### CODE REVIEW

File: .github/workflows/release-generate-ci.yaml [hunk 1]
```diff
@@ -258,6 +306,54 @@ jobs:
...
+          else
+            git push "https://serverless-qe:${GH_TOKEN}@github.com/serverless-qe/$repo.git" "$branch:$branch" -f
+          fi
+          gh pr create --base "$target_branch" --head "serverless-qe:$branch" --title "[$target_branch] Add Konflux configurations" --body "Add Konflux components and pipelines" || true
+        working-directory: ./src/github.com/openshift-knative/hack/openshift-knative/eventing-istio
+      - env:
+          GH_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
+          GITHUB_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
+        if: ${{ (github.event_name == 'push' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule') && github.ref_name == 'main' }}
+        name: '[eventing-kafka-broker - release-next] Create Konflux PR'
```
Comment (creydr) [inline .github/workflows/release-generate-ci.yaml:337]: This seems not correct
/hold

File: config/eventing-kafka-broker.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@ config:
   branches:
     release-next:
       konflux:
+        enabled: true
```
Comment (creydr) [inline config/eventing-kafka-broker.yaml:5]: ?
/hold

Comment (creydr) [reply config/eventing-kafka-broker.yaml:5]: Addressed in https://github.com/openshift-knative/hack/pull/384

File: .github/workflows/release-generate-ci.yaml (hunk 1 above)
Comment (creydr) [reply .github/workflows/release-generate-ci.yaml]: Fixed 
/unhold

File: pkg/konfluxgen/kustomize/docker-build.yaml [hunk 3]
```diff
@@ -86,7 +86,6 @@ spec:
     type: string
   - default:
     - linux/x86_64
-    - linux/arm64
```
Comment (creydr) [inline pkg/konfluxgen/kustomize/docker-build.yaml:89]: from https://github.com/konflux-ci/build-definitions/commit/05d1bbf18510fc4dd91435dc3d9f6dd08e8fa0cf

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**